### ISSUE TITLE
Clustering

### ISSUE THREAD

**Issue by Phoscur** [Naltox/telegram-node-bot#158]
Looks like a cool feature, but really, generating duplicate messages by default sounds rather like a bug :P 
Please explain how to use it correctly and use zero as default worker count!

**Comment by vsviridov** [comment]
You can check something like
```javascript
const client = new Telegram(...);
if(!client._workers)
{
  /* In worker instance */
}
else
{
  /* in master instance */ 
}
```

but having official `client.isMaster` or `client.isWorker` would be nice.

**Comment by AndryFM** [comment]
Is not that what you are looking for? [https://github.com/Naltox/telegram-node-bot#clustering](url)

```
const tg = new Telegram.Telegram('YOUR_TOKEN', {
    workers: 1
})

tg.sendMessage(123, 'test message') //will be sent 2 times (one time on master and one time on worker)

tg.onMaster(() => {
    tg.sendMessage(123, 'test message') //will be sent one time
})
```

**Comment by Phoscur** [comment]
Yes @AndryFM, that's what I'm refering to, and it should answer @vsviridov comment.

What I mean is, that this part is this very much undocumented besides that short paragraph.
It does not explain
* How to actually execute a task (algorithmic work) on worker and then send messages only with master
* How to disable workers

Currently, not reading the refered paragraph will produce weird duplicate messages. The default worker count should be zero, eliminating the need to know about clustering at all.
